portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -75,27 +75,59 @@ public void doDelete(HttpServletRequest request, HttpServletResponse response) t
   /** Returns comments fetched from datastore */
   private ArrayList<Comment> fetchComments(HttpServletRequest request) {
     //Prepare datastore query
-    Query query = new Query("Comment");
-    PreparedQuery results = datastore.prepare(query);
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     
     //Set limit options
     FetchOptions fetchOptions = FetchOptions.Builder.withDefaults();
     Optional<Integer> limit = getLimit(request);
     if (limit.isPresent()) {
       fetchOptions = FetchOptions.Builder.withLimit(limit.get());
     }
+    
+    //Set username filter
```

Comment:
Similar comment to the other PR. Format these comments with one space between `//` and the comment text.